docpairs.js

```diff
@@ -0,0 +1,201 @@
+var fs = require('fs');
```

Comment:
This shouldn't be at the top level of the fathom package, since it isn't intended for reuse. The `examples` folder is probably the right place. You may want to make a `products` folder there and move all the product-related files within.